projects/widgets/docs/inleiding.md

```diff
@@ -0,0 +1,56 @@
+# Widgets
+
+Welkom op onze widget documentatie! 👋
+
+Widgets zijn kant-en-klare HTML/JS modules die je in je website kan integreren om zo snel en eenvoudig een volledig doorzoekbare UiTagenda op je website te publiceren.
+
+publiq biedt een widgetbouwer (wizar) aan waar je gemakkelijk je widget kan configureren, zonder dat je hiervoor geavanceerde technische kennis nodig hebt. Tijdens het configuratieproces worden al je instellingen opgeslagen op een `<script>` tag. Wanneer dit `<script>` tag geïntegreeerd wordt op je website, zal die de geconfigureerde agenda met vrijetijdsactiviteiten weergeven.
```

Comment:
What is wizar?
Or should it be `wizard`

Reply:
Should be `wizard`. Fixed in a new commit